pom.xml

```diff
@@ -611,23 +611,6 @@
     <profiles>
         <profile>
             <id>dev</id>
-            <properties>
```

Comment:
By removing the old dev properties means we do not use the local database anymore, is it? I am wondering if keeping both of dev and dev.local is good in case local database server breaks somehow.

Reply:
The local (Docker or native) database is the one we’re keeping. If you mean `gxpatlasdev` on `pgsql-hxvm-037.ebi.ac.uk`, that one’s going away soon. It reduces the number of environments we need to keep track of and is a step towards containerizing the whole application.